docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -582,6 +582,103 @@ To access the API server from within a Kubernetes cluster, some RBAC related res
 So, when the `kubernetes-client` extension is present, the `kubernetes` extension is going to create those resources automatically, so that application will be granted the `view` role.
 If more roles are required, they will have to be added manually.
 
+=== Customizing RBAC (Cluster)Roles and bindings
+
+Especially for building Kubernetes operators, additional RBAC configuration may be required. The Kubernetes extension got you covered with the `quarkus.kubernetes.policies` config group:
+
+[source,properties]
+----
+# Role and RoleBinding resources are generated and bound to the desired app kind (Deployment or StatefulSet)
+quarkus.kubernetes.policies.ext.api-groups=extensions,apps <1>
+quarkus.kubernetes.policies.ext.resources=deployments,deployments/status
+quarkus.kubernetes.policies.ext.verbs=get,watch,list
+quarkus.kubernetes.policies.srv.resources=services <2>
+quarkus.kubernetes.policies.srv.verbs=get,update
+
+# ClusterRole and ClusterBinding are generated for policy groups defined as 'cluster-wide'
+quarkus.kubernetes.policies.crd.cluster-wide=true <3>
+quarkus.kubernetes.policies.crd.api-groups=apiextensions.k8s.io
+quarkus.kubernetes.policies.crd.resources=customresourcedefinitions
+quarkus.kubernetes.policies.crd.verbs=get,update
+----
+
+<1> `ext` in this example is only a grouping key for identifying a conjugated `PolicyRule` in the `Role` manifest, it's solely used in the configuration file.
```

Comment:
```suggestion
<1> `ext` in this example is only a grouping key for identifying a conjugated `PolicyRule` in the `Role` manifest, its solely used in the configuration file.
```

Reply:
in fact, I wanted to say, that ... the grouping key ... _it is_ solely used...